syntax = "proto2";

message SocketData {
  optional string data_type = 1;
  optional bytes data = 2;
}